### ISSUE TITLE
训练集的选择问题

### ISSUE THREAD

**Issue by taoWuuu** [csguoh/MambaIR#80]
您好，我想问一下为什么经典SR和轻量级SR要更换训练集而不是都使用DF2K，是为了跟其他对比模型采取相同的训练集吗。还有就是请问经典SR的验证集是使用的什么。

**Comment by csguoh** [comment]
Yes, we use DIV2K only for a fair comparison. You can also train a model with DF2K for better performance.
